Clinical-Genomics/cg
# adds possibility to upload many observations in one go

This PR adds option to upload many observations in one go

How to test resetting a loqusdb link in status db:
1. install on stage (add instructions on how to as needed)
1. find a case that has been uploaded to loqus_db
1. `cg reset observations [case_id]`
1. y

Expected outcome:
The loqusdb-links on the samples for the case have been reset 

How to test resetting all loqusdb links in status db:
1. `cg reset observations`
1. y

Expected outcome:
The loqusdb-links on all samples for all cases that are properly internally linked have been reset 

How to test listing cases that are liable for being uploaded to loqus-db:
1. `cg upload observations -p`

Expected outcome:
All families that has not been uploaded to loqusdb should be listed

How to test uploading all observation in one go:
1. `cg upload observations`

Expected outcome:
All families that has not been uploaded to loqusdb should be uploaded
